server/api.proto

```diff
@@ -156,6 +156,11 @@ message Envelope {
     TLeaderboards leaderboards = 57;
     TLeaderboardRecord leaderboard_record = 58;
     TLeaderboardRecords leaderboard_records = 59;
+
+    TMatchmakingStart matchmaking_start = 60;
```

Comment:
OK sounds like the vote says `TMatchmake*` so I'll rename everything in the server, then the client. @mofirouz what about the suffixes? `Start`/`Cancel`/`Ticket`/`Result` or `Add`/`Remove`/`Ticket`/`Matched` or some other combination?